### ISSUE TITLE
Cannot disable Steam Play for supported titles

### ISSUE THREAD

**Issue by eriktorbjorn** [ValveSoftware/steam-for-linux#9957]
#### Your system information

* Steam client version (build number or date): 1690583737, Fri, Jul 28 8:44 PM UTC -08:00
* Distribution (e.g. Ubuntu): Debian (unstable)
* Opted into Steam client beta?: No (not that I'm aware of)
* Have you checked for system updates?: Yes
* Steam Logs: [steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/12269907/steam-logs.tar.gz)
* GPU: AMD (lspci says "VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Bonaire XTX [Radeon R7 260X/360]")

#### Please describe your issue in as much detail as possible:

I opened the Settings dialog and clicked on "Compatibility".

![steam-compat-settings](https://github.com/ValveSoftware/steam-for-linux/assets/601765/e0bf090c-9c3b-403a-be77-071a39a57016)

I clicked on the toggle for "Enable Steam Play for supported titles".

What I expected to happen: The toggle changing to the "off" position.
What actually happened: Nothing.

(The reason I want to disable it is that I don't trust Proton to run well on my hardware, since Vulkan support is experimental.)

Maybe the entire settings page is slightly broken, because this is what the manual says it should look like:

![what-I-am-not-seeing](https://github.com/ValveSoftware/steam-for-linux/assets/601765/924cb993-7d34-41bf-8cd7-b18053161f38)

Steam support were not able to help me with my issue. Actually, I'm not even convinced they read it.

**Comment by eriktorbjorn** [comment]
Possibly related issue: https://github.com/ValveSoftware/steam-for-linux/issues/9419

**Comment by disconnect5852** [comment]
same problem here! Now shows all of my games runnable on linux, while most of them aren't (start it, then nothing ever happens(

**Comment by Matiasgroen** [comment]
Same problem here

**Comment by federicoxd122134** [comment]
Same here, i can't play games natively

**Comment by NetDin0** [comment]
Same here

**Comment by ihatesoftwaresomuch** [comment]
I'm having the same issue. None of my games are working, best case they start but quit before they successfully connect to a server. I'm using the lastest opensuse tumbleweed.

**Comment by bclima** [comment]
Same here....

**Comment by Ligthert** [comment]
Same... :unamused:

**Comment by lucaslira95** [comment]
Same...

**Comment by AlexCloudDev** [comment]
same :(

**Comment by lucifertdark** [comment]
There's a workaround available, you'll need ProtonUp-QT from https://github.com/DavidoTek/ProtonUp-Qt so you can install Steam-Play-None. It will do the job till Valve sort it out properly.

**Comment by goph-R** [comment]
Same, Debian 12, XFCE

**Comment by asif-mahmud** [comment]
i got stuck here too

**Comment by jhahvhi** [comment]
Same
Debian testing gnome

**Comment by nm004** [comment]
I could revert it back by removing the entry of app id subject in `~/.steam/steam/userdata/<steamid>/compat.vdf`.

**Comment by Ligthert** [comment]
I don't know what happened, but in the past couple of weeks this setting has worked for me and I am good. :-)

**Comment by NetDin0** [comment]
I can't believe this bug hasn't been fixed yet.

**Comment by Houtworm** [comment]
This is preventing me from installing native games on my Steam Deck OLED. Very annoying I just want to play native games. It used to work before on my old LCD Deck, If I want a game to run with Proton I will force it. Please fix this issue.

**Comment by her001** [comment]
I did some testing to verify this issue. It is true that there is no off button. However, overriding the compatibility setting to `Steam Linux Runtime 1.0 (scout)` runs the game in the exact same way as a native game without a compatibility tool set. It may be that there is still an issue here, in that this should be made more clear in the UI. Otherwise, the functionality of running the native client for the few games that are set to use Proton by default has not been removed.

Valve [has made it clear](https://github.com/ValveSoftware/steam-runtime#introduction) that the Scout runtime is literally just the libraries shipped with the Steam client, without `pressure-vessel` (the containerization stuff). You can verify this yourself by checking the libraries that games are launched with by Steam:

1. Set the launch options for a native game to `echo $LD_LIBRARY_PATH > ~/libs.txt #%command%`.
2. Run the game.
3. Check the `libs.txt` file in your home folder. For a native game, they will be the same regardless of whether the Scout runtime is selected.
4. You may also check the command in your process list. It is not run with `pressure-vessel` when using Scout.

Further, if you disable the Steam runtime when starting Steam (setting `STEAM_RUNTIME=0`, as with the `steam-native-runtime` package in the Arch repos), setting the compatibility tool to Scout still respects the disabled Steam runtime.

**tldr**: If you want to run the native version of a game, setting the compatibility tool to `Steam Linux Runtime 1.0 (scout)` is the same as unsetting it.

**Comment by Houtworm** [comment]
The issue  is not that something is unclear in the UI, The issue is that a setting in the Steam Settings is broken.

If we untick "Enable Steam Play for all other titles" it should actually do that. not tick it again after a forced reboot.

"Enable Steam Play for supported titles" doesn't toggle at all, and will just stay in the on position if you click it.

Setting Steam Linux Runtime Scout for every game might be a workaround, but it is far from ideal. I don't want to set them all to scout just to turn them all off again when this problem gets fixed.

I don't want Valve to decide how I run my games, I want games to run natively, and preferably display the grayed out install button when there is no Linux depot. 

I want an opt in into proton, not an opt out, this way the line between Proton and Native gets blurred. It is a Linux device so when I press install I expect it to install the Linux build, and if I want to use a compatibility tool I will select one myself.

I hope that this issue will get fixed very soon, I bought a Steam Deck over the Ally for the software, But I might be better off returning the Deck and Install Arch on the Ally while I am still within the return period. At least On Arch I can disable the second toggle.

**Comment by eriktorbjorn** [comment]
> I want an opt in into proton, not an opt out, this way the line between Proton and Native gets blurred.

Me too. Particularly since Proton has never _ever_ actually worked for me when I've tried it. Perhaps because I've never tried it with Vulkan, since that's experimental on my hardware, but it's also never given me any error message about it. It just silently failed.

**Comment by z-impi** [comment]
Same on Arch Garuda.
Button is not moving.
Same in beta. :(

**Comment by LinuxRocks101** [comment]
I'm also having this issue on Linux Mint 21.2 Edge Cinnamon. D:

At least Steam opens in Linux Mint. I had to switch back cause it'd just lockup and never open after the login screen on Kubuntu 23.10. :|

**Comment by flamming-python** [comment]
Even purging the Steam client from my computer, deleting the .steam folder from my home directory, does not seem to reset this option. Valve have really outdone themselves. And this issue is considerably older than this bug report, BTW. Just another bug that has been allowed to go unfixed despite it's pretty serious usability consequences for customers whose computers do not support Proton well or at all

**Comment by LinuxRocks101** [comment]
This **still** not being fixed is so ****Ing annoying. I just wanna organize my damn Steam library properly. If it isn't this, it's Steam not letting me disable Steam Play (the button literally does nothing), or Steam just flat out doesn't even open in Linux and just hangs at the logging in screen... I've got a large Steam library, and being able to organize the games by OS makes my experience easier. 🤬

**Comment by LinuxRocks101** [comment]
> Steam Support told me it's because I'm not using Ubuntu with GNOME... Sure.

What kinda BS operation are you running Valve?

I do tech related testing for a living (both software and hardware) working directly with engineers. I've spent thousands of dollars on Steam, and as such, my rage about this crap with the Steam client made me send a professional yet blunt job application to Valve on their website. Stating my testing experience, why their clients on Linux and Mac suck, and that they obviously need another person to help test them. Since they clearly aren't currently capable of it going off of all the problems and performance issues that there are (especially with those clients). But all I got back from them was a "polite **** off" as a response. lol

**Comment by NoTitleGamesOfficial** [comment]
Same issue using Ubuntu 20.04 LTS

**Comment by MrAventador** [comment]
I have the same issue on Arch

**Comment by NetDin0** [comment]
Today a patch was released, but for me none changed

![image](https://github.com/ValveSoftware/steam-for-linux/assets/43504858/d3bee52a-d9ef-431a-8d42-7b1a9bf43755)

**Comment by kisak-valve** [comment]
Hello @NetDin0, the release note you're highlighting is for #9875. If you're seeing that issue, then follow the instruction at https://github.com/ValveSoftware/steam-for-linux/issues/9875#issuecomment-1853037408 to nudge the game config out of the quirk state.

**Comment by datante** [comment]
Can confirm the issue is still not fixed. I am running Linux Mint 21.3

**Comment by qquq** [comment]
The issue is still not resolved.
Debian 10 LTS
Steam Beta as of date 06.02.2024

**Comment by carlRondoni** [comment]
Not resolved on arch linux :(

**Comment by LinuxRocks101** [comment]
I switched to KDE Neon to try out Plasma 6, and this issue *still* isn't resolved. Will this ever even be looked at and fixed? It can't be that hard or take that long to do. This is probably one of the more important features on Steam in Linux. So you'd think that it'd have been taken care of already. Plus Steam Support was useless about this problem too. 🙄

**Comment by eduardoeae** [comment]
Arch Linux up to date... still not fixed.

**Comment by Hesder** [comment]
I  have the same problem with a fresh install of Ubuntu 24.04. The button works, but has no effect. 

MANUAL WORKAROUND:

Edit the following file and empty the content (This files registers for each game the compatability mode, throwing it away forces steam to generate it again, based on your current settings). 

~/.steam/steam/userdata/115053894/config/compat.vdf

**Comment by Milor123** [comment]
Guys how can disable the fuc***** BETA steam, it crashes my steam :( i dont wanna lost all my data. please help me

**Comment by AngelAcid** [comment]
Problem still exists almost a year later. Pretty good encouragment to not use steam on Linux and take the good ol' black flag approach.

**Comment by zoobporsor** [comment]
I still cant turn it off too. I have no idea if its trying to play Native linux games , or trying to run Proton games. I don't know how to force run native linux games and turn off Proton.

**Comment by nikolaoslibero** [comment]
It seems, as I've seen no communications to the contrary, that with the [October 25th client beta update](https://steamcommunity.com/groups/SteamClientBeta/announcements/detail/4507632758953082943?snr=2___) steam has "it's not a bug, it's a feature"d it.

**Comment by ZEROX7** [comment]
fixed it by exiting steam and deleting the userdata for my account

**Comment by Poyeyo** [comment]
I have this issue, but it was fixed by uninstalling steam, deleting the contents from the .steam folder, and reinstalling it.

So at least there is a workaround.

**Comment by tehdango** [comment]
> It seems, as I've seen no communications to the contrary, that with the [October 25th client beta update](https://steamcommunity.com/groups/SteamClientBeta/announcements/detail/4507632758953082943?snr=2___) steam has "it's not a bug, it's a feature"d it.
> 
> The button was removed entirely.

I really dont like to necro threads but this is sort of crazy and nobody else is really talking about it except here. If steam is forcing steam play to always be enabled then how do we know which of our games has a native linux client? When this is forced on it even runs those games via proton which, I think I speak for all of us, not what we want.....at all.

**Comment by nikolaoslibero** [comment]
<img width="944" height="1044" alt="Image" src="https://github.com/user-attachments/assets/52f0182f-139f-462f-8b4d-bca311932035" />

Apparently, some part of Valve has lost their marbles and removed every option except the default proton version. Gaben, wtf?

Having Steamplay on by default is fine as I imagine it helps Steamdeck users, but good lord are you giving a sh/*t about linux or not? This is f/*cking absurd. This isn't the beta client and this isn't unintentional unless your testing suite is abysmally lacking.

I would love to have a separate library that holds all of the games that run natively, but the penguin button in the library side panel has been mostly broken (it included Windows titles that Steam thinks work well under proton) and now it is ENTIRELY useless as it filters NOTHING.

Is GOG still worth buying from? I don't know that Steam is anymore.

**Comment by nikolaoslibero** [comment]
To confirm, it is indeed on purpose. Steamdeck is giving them brainworms.

[Steam Client Update - June 30th](https://steamcommunity.com/app/593110/eventcomments/599656878323044839?snr=2_groupannouncements_detail_)

**Comment by PimpOfSune** [comment]
> Is GOG still worth buying from?

Depends on what you're looking for.
GOG Galaxy doesn't exist for Linux.
Lutris's GOG integration (and all other integrations) are currently broken. Manual DRM-free install works USUALLY.
Heroic works for GOG, on Linux, natively. However, it needs some optimization, desperately.
And on top of that some native Linux versions which GOG sells are currently broken. ADOM (not Ultimate one) and Victor Vran being among oldest offenders.

There are rumours of PlayNite getting native Linux version this year.
